lang_java/analyze/java_to_generic.ml

```diff
@@ -300,6 +300,10 @@ and expr e =
   | AssignOp ((v1, (v2, tok), v3)) ->
       let v1 = expr v1 and v3 = expr v3 in
       G.AssignOp (v1, (v2, tok), v3)
+  | TypedMetavar (v1, v2, v3) ->
+      let v1 = ident v1 in
+      let v3 = typ v3 in
+      G.TypedMetavar (v1, v2, v3)
```

Comment:
here you can put a fake_parse_info for v2
and later one I'll remove the tok also in AST_generic.TypedMetavar.